### PR TITLE
Fix surefire java version to be built on 1.8

### PR DESCRIPTION
**Pull request by laeubi**

### CODE REVIEW

**Comment by laeubi** [comment]
@akurtakov any idea what could cause the test-failure? I can't reproduce this with a local build and the code-part seems unrelated to surefire or any java compile version...

**Comment by laeubi** [comment]
At laest GH do not shows the failure, local build is also fine...

**Comment by laeubi** [comment]
I added a new PR without changes here ant it fails with the same tests

https://github.com/eclipse-tycho/tycho/pull/1547

but master succeeds:

https://ci.eclipse.org/tycho/job/tycho-github/job/master/

to make progress here I want to disable the test for now to see if anything else fails on Jenkins @akurtakov @mickaelistria any objections?

**Comment by akurtakov** [comment]
I don't see why you have to even ask us for temp disablement of test for experiment in this PR.

**Comment by laeubi** [comment]
> I don't see why you have to even ask us for temp disablement of test for experiment in this PR.

Question is more how long such temp disabling could be used, e.g. I don't see any chance to debug this if it is not reproducible locally, as the test assumes a failure where non seem to happen... so if the test only fails on Jenkins CI but not for anyone else, I fear we need to disable it unless someone finds out how to reproduce this so it can be debugged ... Or we conclude that the test is not that important and remove it permanently.

File: tycho-surefire/pom.xml [hunk 1]
```diff
@@ -14,9 +14,8 @@
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.eclipse.tycho</groupId>
-		<artifactId>tycho-bundles</artifactId>
```
Comment (akurtakov) [inline tycho-surefire/pom.xml:17]: Could this be the issue? I don't see why this change is in this PR and tycho-bundles parent pom overrides target-platform-configuration.

Comment (laeubi) [reply tycho-surefire/pom.xml:17]: Actually tycho-bundles as a parent is not required here as all submodules using non tycho-build (jar packaging) or directly use tycho-bundles as their parent....

Beside that I created a PR without any (code) changes and it also fails with this test :-\ 

See
- https://github.com/eclipse-tycho/tycho/pull/1547
- https://ci.eclipse.org/tycho/job/tycho-github/job/PR-1547/1/

so it is a bit strange ...

**Comment by laeubi** [comment]
I have created https://github.com/eclipse-tycho/tycho/issues/1550 to track enabling the test again.
